docs/content/concepts/assets/graph-backed-assets.mdx

```diff
@@ -58,9 +58,9 @@ def slack_files_table():
 
 ### Defining basic dependencies for graph-backed assets
 
-Similar to with basic software-defined assets, Dagster infers the upstream assets from the names of the arguments to the decorated function.
+Similar to software-defined assets, Dagster infers the upstream assets from the names of the arguments to the decorated function.
 
-The example below includes an asset named `middle_asset`. `middle_asset` depends on `upstream_asset`, and `downstream_asset` depends on `middle_asset`:
+The example below includes a graph-backed asset named `middle_asset`. `middle_asset` depends on `upstream_asset`, and `downstream_asset` depends on `middle_asset`:
```

Comment:
i felt like being more specific that `middle_asset` was the graph backed asset made it easier to find the important part of this example